tests/Functional/ContentNewsRenderTest.php

```diff
@@ -85,6 +85,42 @@ public function testNewsRendering(): void {
     $this->assertEquals('http://www.example.org', $header_link->getAttribute('href'));
     $this->assertEquals('http://www.example.org', $header_link->getText());
 
+    // Assert Sources field content.
```

Comment:
I suppose by switching to implementation based on the `list_item` pattern, we can use the appropriate `assertPattern` method.